src/Sarif/NotYetAutogenerated/SuppressionStates.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.CodeDom.Compiler;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// The state of a result relative to a baseline of a previous run.
+    /// </summary>
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.16.0.0")]
+    [Flags]
+    public enum SuppressionStates
+    {
+        None,
```

Comment:
Shouldn't it be "Unknown"? I thought that was the result of our discussion -- that if you didn't see one of the positive values, that you just couldn't tell if it was suppressed. "None" sounds like it means "not suppressed", which we don't know.